docs/setup-cluster/deploy-cluster/installation-options.rst

```diff
@@ -1,20 +1,135 @@
-#############
- Basic Setup
-#############
+:orphan:
 
-Your training environment can be a local development machine, an on-premise GPU cluster, or cloud
-resources. To set up your training environment, follow the steps below.
+.. _installation-options:
 
-**************************************************
- Step 1 - Set ``DET_MASTER`` Environment Variable
-**************************************************
+######################
+ Installation Options
+######################
+
+In this guide, you'll find information about basic and advanced cluster configuration options, as
+well as additional installation options such as setting up clients, internet access, and firewall
```

Comment:
can we separate client setup into its own page?

My thinking is that in lots of companies, users might get a link to that page from their sysadmin saying, "please follow this page with DET_MASTER=xyz.com" or something, and it would be nice if it were its own page.

Reply:
done